docs/usage/CompatibleOCPVersions.md

```diff
@@ -0,0 +1,20 @@
+# Compatible OCP versions
+
+Refer to the guidelines below to select the correct mig-operator version for your OpenShift cluster version.
+
+## Install version guidelines
+
+ - **OpenShift 3.7 - 4.5**
+   - Install latest Crane 1.5.x
+ - **OpenShift 4.6+**
+   - Install Crane 1.6.x+
+   - Use this as the control cluster (where mig-controller and mig-ui are installed)
+
+_Note_: Crane 1.6.x and Crane 1.5.1 are designed to be compatible when Crane 1.6.x is the control cluster (e.g. the cluster where mig-controller and mig-ui runs).
+
+## Incompatibilities
+
+Due to the switchover from v1beta1 to v1 CRD API in recent versions of OpenShift:
+
+- Crane 1.5.1- is not compatible with OpenShift 4.9+
```

Comment:
@djwhatle 
Does this mean that it is not possible to migrate from OCP 3.7-4.5 to 4.9?

Reply:
1.5.1 **cannot be installed** on 4.9. The supported path to migrate from 3.7-4.5 to 4.9 is to install 1.5.1 onto the former, and 1.6.0 onto the latter.

Reply:
Made a revision to the doc to remove the confusing wording, it's checked into master.